Xamarin.Forms.Core/ListView.cs

```diff
@@ -644,5 +644,25 @@ protected override bool ValidateItemTemplate(DataTemplate template)
 
 			return true;
 		}
+		
+		public IEnumerable<Element> ListViewLogicalChildren => _logicalChildren.AsEnumerable();
+
+		public void AddLogicalChild(Cell content)
+		{
+
+			if (content != null)
+				_logicalChildren.Add(content);
+
+			content.Parent = this;
+			VisualDiagnostics.OnChildAdded(this, content);
```

Comment:
@gentledepp what does this line do? Never seen it 😲

Reply:
which one? "VisualDisgnostics"?
I have no idea - this is essentially copied code from `ListView.SetupContent`  and `ListView.UnhookContent`
which is also why the `Cell` parameter is called "content" ;-)

see here:
```csharp



		protected override void SetupContent(Cell content, int index)
		{
			base.SetupContent(content, index);
			if (content is ViewCell viewCell && viewCell.View != null && HasUnevenRows)
				viewCell.View.ComputedConstraint = LayoutConstraint.None;

			if (content != null)
				_logicalChildren.Add(content);

			content.Parent = this;
			VisualDiagnostics.OnChildAdded(this, content);
		}

		protected override void UnhookContent(Cell content)
		{
			base.UnhookContent(content);

			if (content == null || !_logicalChildren.Contains(content))
				return;
			var index = _logicalChildren.IndexOf(content);
			_logicalChildren.Remove(content);
			content.Parent = null;
			VisualDiagnostics.OnChildRemoved(this, content, index);

		}


```

Reply:
Ok, so be it :)